drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -215,18 +216,16 @@ def studio_view(self, context):
         """
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
-        help_texts = {
-            field_name: self.ugettext(field.help)
-            for field_name, field in self.fields.viewitems() if hasattr(field, "help")
-        }
-        field_values = {
-            field_name: field.values
-            for field_name, field in self.fields.viewitems() if hasattr(field, "values")
-        }
+        # A short random string to append to HTML element ID attributes to avoid multiple instances
+        # of the DnDv2 block on the same page sharing the same ID values.
+        # We avoid using ID attributes in preference to classes, but sometimes we still need IDs to
+        # connect 'for' and 'aria-describedby' attributes to the associated elements.
+        id_suffix = uuid.uuid4().hex[:16]
+        js_templates = js_templates.replace('{{id_suffix}}', id_suffix)
         context = {
```

Comment:
@cahrens js_templates contains Handlebars templates. Replacing `id_suffix` with raw string substitutions is a bit of a hack, however I think it's still better than passing the `id_suffix` variable to JavaScript and making sure it is present in the handlebars context every time when rendering a handlebars template.

We still have to pass the `id_suffix` variable in the context, because we use it inside django templates, too.